page.md

```diff
@@ -0,0 +1,34 @@
+# Conventions for PAGE
+
+## URL for imageFilename / filename
+
+The `imageFilename` of the `<pg:Page>` and `filename` of the `<pg:AlternativeImage>` element MUST be a URL. A local filename should be a `file://` URL.
+
+All URL used in `imageFilename` and `filename` [MUST be referenced in a fileGrp in METS](https://ocr-d.github.io/mets#if-in-page-then-in-mets).
+
+## Original image as imageFilename
+
+The `imageFilename` attribute of the `<pg:Page>` MUST reference the original image and MUST NOT change between processing steps.
+
+## AlternativeImage for derived images
+
+To encode images derived from the original image, the `<pc:AlternativeImage>` should be used. Its `filename` attribute should reference the URL of the derived image.
+
+The `comments` attribute should be one or more (separated by comma) terms of the following list:
+
+## AlternativeImage: classification
+
+The `comments` attribute of the `<pg:AlternativeImage>` attribute should be used
+
+  * `binarized`
+  * `deskewed`
+  * `despeckled`
+  * `cropped`
+  * `rotated-90` / `rotated-180` / `rotated-270`
+  * `dewarped`
+
+## AlternativeImage on sub-page level elements
+
+For the results of image processing that changes the positions of pixels (e.g. cropping, rotation, dewarping), `AlternativeImage` on page level and polygon of recognized zones is not enough to access the image section a region is based on since coordinates are always relative to the original image.
+
+For such use cases, `<pg:AlternativeImage>` may be used as a child of `<pg:TextRegion>`, `<pg:TextLine>` or `<pg:Word>`.
```

Comment:
`<pg:Glyph>`?

Reply:
True, it is specified like that (also for `pg:Glyph`) in the XSD, will add to text.